src/concrete/ml/onnx/ops_impl.py

```diff
@@ -2043,3 +2062,24 @@ def numpy_gather(
     slices = tuple(slice(None) if i != axis else indices_list for i in range(x.ndim))
 
     return (x[slices],)
+
+
+@onnx_func_raw_args("shape")
+def numpy_expand(x: numpy.ndarray, shape=None) -> Tuple[numpy.ndarray]:
```

Comment:
shape parameter is missing a typing (I guess `Optional[Tuple[int]]`) !

Reply:
Indeed. But this is true for many of our numpy ops actually (i copied pasted from unsqueeze).

I will add a commit to fix the typings maybe.

Reply:
yeah not surprised that some old stuff are not up to date with our typing / doc conventions !

Reply:
Done